test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Data.IO;
 using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Recommender;
```

Comment:
why we getting this namespace?
And can you move MF test into Training.cs file? since it's about training and everything here is about transforms.

Reply:
It is using static MF and Pete prefers this location.

Reply:
Let's ask him @zruty0

---
In reply to: [229069437](https://github.com/dotnet/machinelearning/pull/1407#discussion_r229069437) [](ancestors = 229069437)

Reply:
And speaking about internal MF predictor, it's not internal anymore if you expose it in API, so it should live in Microsoft.ML.Trainers namespace

---
In reply to: [229078577](https://github.com/dotnet/machinelearning/pull/1407#discussion_r229078577) [](ancestors = 229078577,229069437)

Reply:
Do you mean moving MFPredictor to ML.Trainers?

Reply:
Moved although it will be changed again soon...

---
In reply to: [229090140](https://github.com/dotnet/machinelearning/pull/1407#discussion_r229090140) [](ancestors = 229090140)